client/js/components/NavigationDrawer.vue

```diff
@@ -44,9 +46,8 @@ export default {
     };
   },
   computed: {
-    authenticated() {
-      return this.$store.state.auth.authenticated;
-    },
+    // TODO remove auth, move to middleware
+    ...mapGetters(['authenticated']),
```

Comment:
Moved to a getter. There's probably more places to move this to but this is fine for now.